File: python/projects/project04/models/motoboy.py
from models.employee import Employee

class Motoboy:
    def __init__(self, funcionario: Employee, cnh: str) -> None:
        self.funcionario = funcionario
        self.cnh = cnh

    def __str__(self) -> str:
        return (
                f"\nMotoboy"
                f"\n{self.funcionario}"
                f"\nCNH: {self.cnh}"
                )

File: java/object-oriented programming/encapsulation/encapsulation13/motoboy.java
package com.mycompany.mavenproject3;

public class Motoboy extends Funcionario{
    private String cnh;

    public Motoboy(String cnh, String nome, String cpf, String rg, double salario) {
        super(nome, cpf, rg, salario);
        this.cnh = cnh;
    }

    public String getCnh() {
        return cnh;
    }

    public void setCnh(String cnh) {
        this.cnh = cnh;
    }

    @Override
    public String toString() {
        return "\nMotoboy" + 
               super.toString() +
               "\nCnh: " + cnh;
    }
}
